server/room-battle.ts

```diff
@@ -1593,6 +1593,7 @@ export class BestOfGame extends RoomGame<BestOfPlayer> {
 			}
 			this.room.add(Utils.html`|html|${winner.name} won game ${this.games.length}!`).update();
 			if (winner.wins >= this.winThreshold) {
+				this.setEnded();
```

Comment:
Shouldn't this be inside `onEnd`?

Reply:
oh without a shadow of a doubt this should be added